long_text_writer.py

```diff
@@ -0,0 +1,48 @@
+from constants import *
+
+class LongTextWriter(pygame.sprite.Sprite):
+
+    def __init__(self, long_text, font='courier', size=16, color='white', width=300):
+        super().__init__()
+        self.text = long_text
+        self.font = pygame.font.SysFont(font, size)
+        self.size = size
+        self.width = width + 100
```

Comment:
Yes.  The +6 on the self.line_height is so that there is padding above and below a line of text.  The +100 on the self.width is to ensure that we see the entire line of text and allow for some padding to the left and right.  Width by itself is to text wrap a line so that the length doesn't go too far beyond the width.  I will change that line to use the variable.

Reply:
Ok cool, sounds good :)